packages/core/src/functions/slack/notifyModeration.ts

```diff
@@ -42,7 +43,7 @@ export const notifyModeration = inngest.createFunction(
             fields: [
               {
                 type: "mrkdwn",
-                text: `*Chat*: <https://labs.thenational.academy/aila/${args.chatId}|aila/${args.chatId}>`,
+                text: `*Chat*: <https://${process.env.VERCEL_URL}/aila/${args.chatId}|aila/${args.chatId}>`,
```

Comment:
This won't be the labs dot domain, it'll be the ...vercel.app - will that cause a clerk issue?

Reply:
Good thinking, have amended